README.md

```diff
@@ -75,21 +75,15 @@ Configure the MESH API connection using the following environment variables:
 | LAB_RESULTS_MESH_ENDPOINT_CERT                 |         | The content of the PEM-formatted client endpoint certificate
 | LAB_RESULTS_MESH_ENDPOINT_PRIVATE_KEY          |         | The content of the PEM-formatted client private key
 | LAB_RESULTS_MESH_SUB_CA                        |         | The content of the PEM-formatted certificate of the issuing Sub CA. Empty if LAB_RESULTS_MESH_CERT_VALIDATION is false
-| LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS |         | (*) The mapping between each recipient HA Trading Partner Code (HA Link Code) to its corresponding MESH Mailbox ID mapping. There is one mapping per line and an equals sign (=) separates the code and mailbox id. For example: "COD1=A6840385\nHA01=A0047392"
+| LAB_RESULTS_MESH_RECIPIENT_MAILBOX_ID_MAPPINGS |         | (*) The mapping between each NHSACK recipient to its corresponding MESH Mailbox ID mapping. There is one mapping per line and an equals sign (=) separates the code and mailbox id. For example: "000000004400001=A6840385\000000024600002=A0047392"
 | LAB_RESULTS_SCHEDULER_ENABLED                  | true    | Enables/disables automatic MESH message downloads
 
-(*) The three-character "Destination HA Cipher" required for each outbound API request uniquely identifies that patient's 
-managing organisation. Each managing organisation also has a four-character "HA Trading Partner Code" (HA Link Code) uniquely
-identifying that patient's managing organisation for the purpose of EDIFACT messaging. Finally, each "HA Trading Partner Code"
-is assigned a MESH Mailbox ID: the mailbox to which the EDIFACT files for a given recipient are sent. The mappings between
-organisations' "HA Trading Partner Codes" and their MESH Mailbox IDs are controlled by this variable. Note: A "Destination HA Cipher" 
-can usually be converted into a "HA Link Code" by appending 1 or 01 to create the four-character code. If in doubt consult 
-with the operator of the LAB_RESULTS instance for the correct value.
+(*) 15 digit NHS Sender ID required for successful NHSACK dispatch
 
 The following three variables control how often the adaptor performs a MESH polling cycle. During a polling cycle the 
 adaptor will download and acknowledge up to "the first 500 messages" (a MESH API limit).
 
-Important: If the MESH mailbox uses workflows other than `LAB_RESULTS_REG` and `LAB_RESULTS_RECEP` then these messages must be
+Important: If the MESH mailbox uses workflows other than `PATH_MEDRPT_V3` then these messages must be
```

Comment:
I don't think we've added this workflow yet to the adapter, we're still supporting `LAB_RESULTS_REG` and `LAB_RESULTS_RECEP`

Reply:
Discussed yesterday